src/Core/System/Test/SystemConfig/Command/ConfigSetCommandTest.php

```diff
@@ -0,0 +1,70 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Test\SystemConfig\Command;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\IntegrationTestBehaviour;
+use Shopware\Core\System\SystemConfig\Command\ConfigSet;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
+use Shopware\Core\Test\TestDefaults;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class ConfigSetCommandTest extends TestCase
+{
+    use IntegrationTestBehaviour;
+
+    private ConfigSet $configSetCommand;
+    private SystemConfigService $systemConfigService;
+
+    protected function setUp(): void
+    {
+        $this->systemConfigService = $this->createMock(SystemConfigService::class);
+        $this->configSetCommand = new ConfigSet($this->systemConfigService);
+    }
+
+    public function configValueProvider(): iterable
+    {
+        /* config_key, value, expected_value, decode */
+        yield 'String false' => ['fake_config_key', 'false', 'false', false];
```

Comment:
the `key` value is always the same, you can hard code it in the testCase itself, that way the test data provider get's denser and is easier to understand